src/core/math/vec3/min.test.js

```diff
@@ -0,0 +1,48 @@
+const test = require('ava')
+const {min, fromValues} = require('./index')
+
+test('vec3: min() called with two parameters should return a vec3 with correct values', t => {
```

Comment:
was min not removed from the index ? or was that just commented out for a while ?